server.go

```diff
@@ -541,7 +506,7 @@ func (s *Server) Restart() {
 	}
 
 	// Ensure we never block on the restartChan, if we're in the middle of a
-	// setup or teardown process we won't be listening on this channel and if so
+	// setup or tare-down process we won't be listening on this channel and if so
```

Comment:
Is it really called this? Never heard it called "tare-down" and don't find anything on it.

Reply:
My editor marked that as a misspell, it is obviously not! Will fix!

- [ ] Revert teardown documentation "Fix".